heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -43,15 +48,23 @@ export interface IJestPluginOptions {
 
 export interface IHeftJestConfiguration extends Config.InitialOptions {}
 
+const PLUGIN_NAME: string = 'JestPlugin';
+const PLUGIN_SCHEMA_PATH: string = `${__dirname}/schemas/heft-jest-plugin.schema.json`;
```

Comment:
Use `path.json()`.  On Windows `__dirname` has backslashes which would produce `C:\path\to/schemas/heft-jest-plugin.schema.json`